runtime/compiler/p/codegen/PPCPrivateLinkage.cpp

```diff
@@ -2738,29 +2739,69 @@ void buildCRC32CCall(TR::Node *callNode,
       deps->getPostConditions()->setDependencyInfo(map.getTargetIndex(TR::RealRegister::gr7), wasteArg, TR::RealRegister::gr7, UsesDependentRegister);
       }
 
-   TR::Register *gr0Reg = deps->searchPreConditionRegister(TR::RealRegister::gr0);
-   TR::Register *gr11Reg = deps->searchPreConditionRegister(TR::RealRegister::gr11);
-   TR::Register *gr12Reg = deps->searchPreConditionRegister(TR::RealRegister::gr12);
+   if (comp->compileRelocatableCode())
```

Comment:
For non-relocatable code we choose between `crc32_vpmsum` and `crc32_no_vpmsum` based on

```
comp->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P8) &&
comp->target().cpu.supportsFeature(OMR_FEATURE_PPC_HAS_VSX)
```

However, for relocatable code we will always call `crc32_vpmsum`. What happens if the runtime processor doesn't support it?

Reply:
`buildCRC32CCall` is under the same flags. I removed these checks inside the function.
I also tested for incompatibility of processors in case of Portable SCC. The AOT headers will mismatch and hence not use compiled method from SCC.